### PR TITLE
latest version but without weather

### PR DESCRIPTION
**Pull request by zanetton**
Trying to submit

### CODE REVIEW

**Comment by ptolemybarnes** [comment]
:checkered_flag:

File: lib/airport.rb [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+   def approves_take_off
+     fail 'No planes at airport' if empty?
+		 planes.pop
+     end
+
+     def stormy?
+       false
+     end
+
+private
```
Comment (ptolemybarnes) [inline lib/airport.rb]: Have a look at Array#delete, which removes a specific object from an array, not just the last one.

Comment (zanetton) [inline lib/airport.rb]: Yes meant to amend that! Can I just do array. Delete and it picks the right one!?! Or how to a specify the plane?

+44 774 0933309

File: lib/plane.rb [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+    def flying?
+      @flying
+    end
+
+    def landed
+	     @flying = false
+    end
+
+    def take_off
+      @flying = true
```
Comment (ptolemybarnes) [inline lib/plane.rb]: Good. But be careful about what your methods return. This #take_off method will return 'true'. What error would you get if you called plane.take_off.flying?

File: spec/plane_spec.rb [hunk 3]
```diff
@@ -1,25 +1,37 @@
...
-  xit 'has a flying status when in the air'
+   it 'has a flying status when not at an airport'do
+   plane = Plane.new
+   expect(plane).to be_flying
+end
 
-  xit 'can take off'
+	it 'can be landed'do
+	plane =Plane.new
+  airport = Airport.new
```
Comment (ptolemybarnes) [inline spec/plane_spec.rb]: Careful with your indentation here.. makes it very hard to read.

**Comment by ptolemybarnes** [comment]
:checkered_flag:

**Pull request closed without merging**